crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -129,16 +124,14 @@ pub struct RealRibosome {
     /// Dependencies for every zome.
     pub zome_dependencies: Arc<HashMap<ZomeName, Vec<ZomeIndex>>>,
 
-    pub serialized_module_cache: Arc<RwLock<SerializedModuleCache>>,
-
-    pub instance_cache: Arc<RwLock<InstanceCache>>,
+    /// File system and in-memory cache for wasm modules.
```

Comment:
instance cache has been removed. instead a deserialized module cache has been added. Both serialized_module_cache and deserialized_module_cache have been abstracted to module_cache and moved to `holochain_wasmer`.